ArmEmulator/MemTests.fs

```diff
@@ -567,47 +568,53 @@ module MemTests
                                     EQUExpr = (Some (Ok valO)); DCDValueList = None;
                                     FillN = None})
         let makeLabelInstr label root input =
-            let ldFuncEQU Lab Ops = 
+            let ldFuncEQU lab ops  = 
                     {LoadAddr= WA 100u; 
-                        Label= Some Lab; 
+                        Label= Some lab; 
                         SymTab= Some stOneItem;
                         OpCode= "";
-                        Operands= Ops}
-            let removeRecord x =
-                match x with
-                | Ok y -> y 
+                        Operands= ops}
             ldFuncEQU label input
             |> parseLabelIns root
-            |> removeRecord 
         let makeTest symTab inpRec field name output =
-            testCase name <| fun () ->
-                Expect.equal (updateSymbolTable symTab inpRec field) output (sprintf "checkUpdateSymbolTable Tests\nTest: %A" name)
+            match inpRec with
+            | Ok v ->   match field with 
+                        | "EQU" ->  testCase name <| fun () ->
```

Comment:
If you start a new line after the `->` you can just indent and the line doesn't end up so long